pages/resultset.py

```diff
@@ -12,6 +12,10 @@
 
 class ResultsetPage(Page):
 
+    _page_title = '[0] mozilla-inbound'
+
+    _first_resultset_locator = (By.CSS_SELECTOR, '.th-view-content > ng-view:nth-child(1) > div:nth-child(2) > div:nth-child(1) > span:nth-child(1) > span:nth-child(1) > span:nth-child(2) > a:nth-child(1)')
```

Comment:
I think we should name this `_resultset_link_locator` and the following would be a less fragile locator: `.result-set .result-set-title-left > span a`.